api/src/clojure/org/akvo/flow_api/unilog/unilog.clj

```diff
@@ -104,32 +105,32 @@
       []
       reducible)))
 
-(defn after-forms-loaded [{::keys [forms-instances-grouped-by-form
-                                   unilog-id
-                                   form-updated
-                                   form-deleted
-                                   form-instance-deleted
-                                   data-point-changed
-                                   data-point-deleted
-                                   survey-changed
-                                   survey-deleted]}
-                          form-id->form]
+(defn filter-events-by-authorization [{::keys [forms-instances-grouped-by-form
+                                               unilog-id
+                                               form-updated
+                                               form-deleted
+                                               form-instance-deleted
+                                               data-point-changed
+                                               data-point-deleted
+                                               survey-changed
+                                               survey-deleted]}
+                                      form-id->form
+                                      authorized-forms
```

Comment:
No see the need of both the authorised-forms and the form-id->form. As it is `authorised-forms == (keys form-id->form)`